calibration_matrix: [-0.9998634,0.01652385,0,-0.5704231,
-0.01652385,-0.9998634,0,0.1234381,
0,0,1,0,
0,0,0,1]